### PR TITLE
add Auth option

### LINKED ISSUE #2: Custom authentication
We should support custom options to be passed to each `nes.Client`

### PR DESCRIPTION
**Pull request by thekemkid**
fixes #2

### CODE REVIEW

**Comment by davidmarkclements** [comment]
@mcollina happy with this?

File: package.json [hunk 1]
```diff
@@ -28,7 +28,9 @@
   },
   "homepage": "https://github.com/nearform/anger#readme",
   "devDependencies": {
+    "bcrypt": "^0.8.7",
```
Comment (mcollina) [inline package.json]: I don't think we need bcrypt for tests, just use plain strings.

**Comment by mcollina** [comment]
Just a nit, you can merge afterwards.

File: package.json (hunk 1 above)
Comment (thekemkid) [inline package.json]: Good idea, will fix

**Pull request merged**